great_expectations/core/configuration.py

```diff
@@ -15,6 +15,17 @@ def __init__(self, id_: Optional[str] = None, name: Optional[str] = None) -> Non
         self.name = name
         super().__init__()
 
+    @classmethod
+    def _dict_round_trip(cls, schema: Schema, target: dict) -> dict:
```

Comment:
Could we make this public? Especially since it is being invoked outside the class?

Reply:
@anthonyburdi  and I talked about making this private because we hope to make this unnecessary with some follow-up work.

Reply:
works for me